images/Dockerfile.test

```diff
@@ -0,0 +1,13 @@
+from golang:1.11-alpine3.8 as runner
+workdir /go/src/github.com/cloudfoundry-incubator/blockhead
+copy . .
+
+RUN apk add --update nodejs nodejs-npm
+RUN apk add git gcc libc-dev python make g++
+RUN go get github.com/golang/protobuf/proto
+
+RUN npm install --production --quiet cli-flags@2.0.7
+RUN npm install --production --quiet web3@1.0.0-beta.36
+RUN npm install --production --quiet solc@0.4.25
+
+cmd ["go", "test", "-v", "./..."]
```

Comment:
thats a good idea.

Reply:
thats what I tried initially and the problem is the makefile needs to shell out to get the current directory and doing a src copy appeared to be slightly cleaner. This would also make the top layers be cached, so i think we should be ok.